Gems/Atom/Tools/ShaderManagementConsole/Code/Source/main.cpp

```diff
@@ -27,19 +28,10 @@ int main(int argc, char** argv)
     QApplication::setOrganizationDomain("o3de.com");
     QApplication::setApplicationName("O3DE Shader Management Console");
```

Comment:
Can these lines be deleted now that they are in the base application and application class?
QApplication::setOrganizationName("O3DE");
    QApplication::setOrganizationDomain("o3de.com");
    QApplication::setApplicationName("O3DE Shader Management Console");